README.md

```diff
@@ -160,90 +159,38 @@ stman -help
 ```
 According to the configured boot mode, place the OS package(s) at the STDATA partition of the stboot image or upload it to a provisioning server. See [Boot Modes](#Boot-Modes)
 
-# Deployment
-Regarding deployment, we defined three real-world scenarios which should at least support a high chance that we have covered a lot of hardware systems. The installation option can be set via `ST_INSTALLATION_OPTION` in `.config`.
```

Comment:
Although we do not have MBR anymore, schouldn;t we say a few words about deployment?
That it runs as efi app, what to do with ISO/IMG

And maybe also leave the coreboot WIP in?

Reply:
Hmm, we could add this information to the general part how stboot works.
In this PR i only removed obsolete/not up-to-date info.
Instead of having a variable to define the installation option, we could also add the coreboot version as task target.

Reply:
let's address this in a separate PR